_posts/daiblej/2020-09-02-first-reflection-daiblej

```diff
@@ -0,0 +1,11 @@
+---
+layout:post
+author:daiblej
+title:"Janet's first class reflection"
+---
```

Comment:
Same comment RE needing spaces after the `:`s.

Also, this filename should have a `.md` on the end or our blog won't know it's supposed to be a post.

Reply:
Thank you. I should be more careful.